charts/cmc-citizen-frontend/Chart.yaml

```diff
@@ -13,16 +13,16 @@ maintainers:
 dependencies:
   - name: nodejs
     version: 2.4.3
-    repository: '@hmctspublic'
+    repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
```

Comment:
are these changes part of this Jira CIV-446? doesn't seems like related to the story

Reply:
This occurred due to Jenkins with this aea19c31d. @tamir-b you would need to fix them in yarn-audit-known-issues